tests/integration_tests/security/test_spectre.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Tests ensuring SPECTREv2 mitigations are present."""
+
+import pytest
+import platform
+
+from framework import utils
+from framework import defs
+
+
+@pytest.mark.timeout(50)
+@pytest.fixture(autouse=True)
```

Comment:
The fixture should have session scope. We only need to run it once per test session. 

```
@pytest.fixture(autouse=True, scope="session")
```

With those 2 things, and if you move it to `tests/integration_tests/performance/conftest.py`, I think you then don't need to make any changes anywhere else (no need to request the fixture explicitly), but then it will run for sessions. And it would only run once **iff** the performance tests are run.